control/plugin/plugin_deprecated.go

```diff
@@ -0,0 +1,308 @@
+/*          **  DEPRECATED  **
```

Comment:
We probably want this separate from the copyright notice separated by a line in it's own comment block. We want attention to be drawn to it.